gpu_ocean/SWESimulators/SimWriter.py

```diff
@@ -47,6 +48,10 @@ def __init__(self, sim, filename=None, num_layers=1, staggered_grid=False, \
                  ignore_ghostcells=False, \
                  offset_x=0, offset_y=0):
 
+        # Parallel netCDF4 write?
```

Comment:
I don't think we will ahve time to support parallel write - so perhaps remove all of the if (write_parallel)-stuff?

Reply:
I agree. I will remove it if/when it requires maintenance.